controllers/cluster_controller.go

```diff
@@ -596,6 +596,10 @@ func getStartCommandLines(cluster *fdbtypes.FoundationDBCluster, processClass st
 		fmt.Sprintf("locality_zoneid = %s", zoneVariable),
 	)
 
+	if processClass == "fast_restore" {
+		confLines = append(confLines, "blob_credential_file = /var/secrets/blob_credentials.json")
```

Comment:
Do we want to hardcode this, or is it better to make it a configuration option? It feels weird to hardcode a path that the operator itself doesn't provide. It may also make it less clear that users need to define the volume mount for the secrets themselves.